### ISSUE TITLE
[SEMS] CANCEL is not being forwarded to b-leg

### ISSUE THREAD

**Issue by denyspozniak** [sems-server/sems#99]
Hello!
Probably I detected issue with next_hop call rerouting.
SEMS does not forward CANCEL message to b-leg.

![image](https://user-images.githubusercontent.com/22612345/50483425-6c96c080-09f4-11e9-884b-e39c12f8bdbf.png)

Sbcprofile settings:
...
next_hop=10.10.10.10/UDP,10.10.10.20/UDP
patch_ruri_next_hop=yes
next_hop_for_replies=yes

**Comment by juha-h** [comment]
I using this kind of simpler next hop config and cancel was properly forwarded to b leg:

next_hop=10.10.10.10/UDP,10.10.10.20/UDP
next_hop_1st_req=no

Could you try with that?

**Comment by denyspozniak** [comment]
Probably issue was resolved by patch from #98 as I could not reproduce it on master branch.
By the way **next_hop_1st_req** is **no** by default:

```
# optional parameter: next_hop_1st_req={yes|no}
# - if set to yes, next_hop behavior (routing without pre-loaded route set)
#   applies only to initial request.  Subsequent requests are routed
#   normally based on route set learned from reply to initial request.
#
#   default: no
#
# Example:
#   next_hop_1st_req=yes

```
